### ISSUE TITLE
Screen reader not reading properly results panel of dept

### ISSUE THREAD

**Issue by GarvinH** [TBS-EACPD/infobase#901]
Specifically this panel

![image](https://user-images.githubusercontent.com/25855114/108074870-acd3b580-7037-11eb-84a0-7759cd497542.png)

Tabbing into core responsibility from the top on chrome reads "collpase this panel.... GCInfobase..." and doesn't read the definition in the end. Tabbing backwards into it works fine however.

A more general problem in the code: tabbing into the status filter reads for each one `Activate to filter "[object Object]"` which happens in the non-a11y version only.
The "Text" that is passed into the status filter for the non-a11y version is actually a group of elements (presumably a span?).

One option we can take is to split the text prop into two props (render and sr-only props). There might be a better way of handling it so I'll leave that up in the air.

**Comment by ktw1016** [comment]
Just tried it and reads correctly `Activate to filter "Met"...`, I'm guessing this was fixed in the PR
